catroid/src/androidTest/java/org/catrobat/catroid/test/ProjectManagerTest.java

```diff
@@ -172,4 +174,46 @@ public void testLoadProjectWithInvalidNestingBrickReferences() throws IOExceptio
 
 		TestUtils.deleteProjects(PROJECT_NAME_NESTING_BRICKS);
 	}
+
+	@Test
+	public void testMergeProjectWithNoConflicts() throws IOException {
+		Project project1 = TestUtils.createProjectWithLanguageVersion(CURRENT_CATROBAT_LANGUAGE_VERSION,
+				"testProject1");
+		Project project2 = TestUtils.createProjectWithLanguageVersion(CURRENT_CATROBAT_LANGUAGE_VERSION,
+				"testProject2");
+		Scene sceneOne = new Scene("test1", project1);
+		Scene sceneTwo = new Scene("test2", project2);
+
+		project2.getSceneByName("Scene 1").setName("Scene 2");
+
+		project1.addScene(sceneOne);
+		project2.addScene(sceneTwo);
+
+		XstreamSerializer.getInstance().saveProject(project1);
+		XstreamSerializer.getInstance().saveProject(project2);
+
+		Project mergedProject = ProjectManager.mergeProjects("testProject3", project1, project2);
+		assertEquals("testProject3", mergedProject.getName());
```

Comment:
this merely asserts that there is a new project with name "testProject3". it could be empy and this test would not fail. 
one could replace the actual piece of code under test, namely ProjectManager.mergeProjects("testProject3", project1, project2); with a simple new Project("testProject3") and this test would still be green.